superset-frontend/src/SqlLab/main.less

```diff
@@ -259,6 +257,14 @@ div.Workspace {
     &:active {
       background: none;
     }
+
+    svg {
+      vertical-align: middle;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
```

Comment:
Instead of this, you can add the styles in JS as follows:
```tsx
const CloseIcon = styled(Icon)`
  vertical-align: middle;

  &:hover {
    cursor: pointer;
  }
`;

// then inside the render function do
<CloseIcon role="button" tabIndex={0} name="cancel-x" onClick={() => this.removeQueryEditor(qe)} />
```

Reply:
If every time use our own svg need this much trouble, i would rather use Font-Awesome, they took care of these troubles already

Reply:
I think you can just add a prop to the `Icon` component and extend the style there.

Reply:
Actually, you don't have to do anything. Just apply `cursor` directly to `CloseIcon` since it's a valid attribute for SVG elements. `<CloseIcon role="button" cursor="pointer">`.

You also don't have to `&:hover` for cursor attributes, since it already only applies when an element is hovered.

Reply:
but I still need `vertical-align: middle;`

Reply:
Yeah, I think it makes sense to keep that rule in `main.less` for now. We'll see if it's necessary to generalize it to `Icon.tsx` when we see it being needed in more places.

Reply:
ok, i will follow this suggestion :)